### PR TITLE
[ES-6974] adding README to all example games

### PR DESCRIPTION
**Pull request by aaron-skillz**

### CODE REVIEW

File: 2048-Android/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+* Danqing Liu
+* Scott Matthewman
+* Sihao Lu
+
+Original code: https://github.com/danqing/2048
+
+Use governed by the MIT License.
+
+## Build Environment
+This project is integrated the Skillz SDK version 25.0.25. Check the [Downloads](https://developers.skillz.com/downloads) page for the latest version, and the instructions are located [here](https://docs.skillz.com/docs/installing-skillz-android/)
```
Comment (jclin) [inline 2048-Android/README.md]: Typo, the version should be `25.0.22`

File: 2048-iOS-Objc-C/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* Danqing Liu
+* Scott Matthewman
+* Sihao Lu
+
+Original code: https://github.com/danqing/2048
+
+Use governed by the MIT License.
+
+## Build Environment
+This project is integrated the Skillz SDK version 25.0.25. Check the [Downloads](https://developers.skillz.com/downloads) page for the latest version, and the instructions are located [here](https://docs.skillz.com/docs/installing-skillz-ios/)
```
Comment (jclin) [inline 2048-iOS-Objc-C/README.md]: Same thing here

File: 2048-iOS-Swift/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+* Scott Matthewman
+* Sihao Lu
+
+Original code: https://github.com/danqing/2048
+
+Use governed by the MIT License.
+
+## Build Environment
+
+This project is integrated the Skillz SDK version 25.0.25. Check the [Downloads](https://developers.skillz.com/downloads) page for the latest version, and the instructions are located [here](https://docs.skillz.com/docs/installing-skillz-ios/)
```
Comment (jclin) [inline 2048-iOS-Swift/README.md]: And here.

**Comment by jclin** [comment]
Just need to use `25.0.22` as the SDK version in all the READMEs.

**Review by jclin — CHANGES_REQUESTED** [review]
Just need to update all the READMEs to use `25.0.22` as the SDK version.

**Comment by trig-skillz** [comment]
👍

**Review by jclin — APPROVED** [review]
:shipit:

**Pull request merged**